src/browser/src/app/components/extension-select/push-api/push-api.component.ts

```diff
@@ -28,16 +29,18 @@ export class PushApiComponent implements OnInit {
 
   ngOnInit(): void {
     this.initData();
+    this.watchInstalledExtensionsChange();
     this.messageService
       .get()
       .pipe(takeUntil(this.destroy$))
-      .subscribe((inArg: Message) => {
-        if (inArg.type === 'extensionsChange') {
-          this.initData();
-        }
-      });
+      .subscribe((inArg: Message) => {});
+  }
+  @ExtensionChange('pushAPI')
+  watchInstalledExtensionsChange() {
+    this.initData();
   }
   initData = () => {
+    console.log('pushApi执行');
```

Comment:
![image](https://user-images.githubusercontent.com/15225835/226526314-b14414a3-1641-48aa-9442-64dfeb91b0e7.png)
featureName can extract to constant